### ISSUE TITLE
GTMSessionFetcher/Core dependency incompatibility when trying to run pod install and also have new firebase libraries installed

### ISSUE THREAD

**Issue by dnkoulouris** [google-ar/arcore-ios-sdk#65]
Firebase GTMSessionFetcher/Core version: 2.1  
ARCore GTMSessionFetcher/Core version: 1.2
```
[!] CocoaPods could not find compatible versions for pod "GTMSessionFetcher/Core":
  In Podfile:

ARCore/CloudAnchors (~> 1.34.0) was resolved to 1.34.0, which depends on
        GTMSessionFetcher/Core (~> 1.2)

Firebase/Storage (= 10.0.0) was resolved to 10.0.0, which depends on
        FirebaseStorage (~> 10.0.0) was resolved to 10.0.0, which depends on
          GTMSessionFetcher/Core (~> 2.1)
```

**Comment by dnkoulouris** [comment]
any progress on this?

**Comment by jrullman** [comment]
This should be resolved by upgrading to ARCore 1.35.0.
